### PR TITLE
Updated example for new cache statistics API

### PR DESCRIPTION
**Pull request by jdrossl**
https://github.com/craftercms/craftercms/issues/1999

### CODE REVIEW

File: source/developers/projects/engine/api/site/cache/statistics.rst [hunk 1]
```diff
@@ -42,18 +42,17 @@ Response
...
-        "localHeapSize": 147,
-        "localOffHeapSize": 0,
-        "localDiskSize": 0,
-        "remoteSize": 0,
-        "writerQueueLength": 0,
-      }
-    }
-  ]
+  {
+    "size": 17,
```
Comment (sumerjabri) [inline source/developers/projects/engine/api/site/cache/statistics.rst]: I thought `size` was too expensive of a call? Is that size in number of objects or bytes?

File: source/developers/projects/engine/api/site/cache/statistics.rst [hunk 2]
```diff
@@ -42,18 +42,17 @@ Response
...
+    "localHeapSize": 17,
```
Comment (sumerjabri) [inline source/developers/projects/engine/api/site/cache/statistics.rst]: What's the unit here?

File: source/developers/projects/engine/api/site/cache/statistics.rst [hunk 3]
```diff
@@ -42,18 +42,17 @@ Response
...
+    "localOffHeapSize": 0,
```
Comment (sumerjabri) [inline source/developers/projects/engine/api/site/cache/statistics.rst]: Unit?

File: source/developers/projects/engine/api/site/cache/statistics.rst [hunk 4]
```diff
@@ -42,18 +42,17 @@ Response
...
+    "localDiskSize": 0,
```
Comment (sumerjabri) [inline source/developers/projects/engine/api/site/cache/statistics.rst]: Unit?

File: source/developers/projects/engine/api/site/cache/statistics.rst [hunk 5]
```diff
@@ -42,18 +42,17 @@ Response
...
+    "cacheHitRatio": 1,
```
Comment (sumerjabri) [inline source/developers/projects/engine/api/site/cache/statistics.rst]: What's the ratio's unit? Should it be a fraction of sorts?

File: source/developers/projects/engine/api/site/cache/statistics.rst (hunk 1 above)
Comment (jdrossl) [reply source/developers/projects/engine/api/site/cache/statistics.rst]: that is correct, the expensive one is the size in bytes

File: source/developers/projects/engine/api/site/cache/statistics.rst (hunk 2 above)
Comment (jdrossl) [reply source/developers/projects/engine/api/site/cache/statistics.rst]: same as size, number of objects

File: source/developers/projects/engine/api/site/cache/statistics.rst (hunk 5 above)
Comment (jdrossl) [reply source/developers/projects/engine/api/site/cache/statistics.rst]: the Ehcache API returns a `double ` for this field, keep in mind this in my local with a single session so the cache is barely used

File: source/developers/projects/engine/api/site/cache/statistics.rst (hunk 3 above)
Comment (jdrossl) [reply source/developers/projects/engine/api/site/cache/statistics.rst]: same as size, number of objects

File: source/developers/projects/engine/api/site/cache/statistics.rst (hunk 4 above)
Comment (jdrossl) [reply source/developers/projects/engine/api/site/cache/statistics.rst]: same as size, number of objects

**Pull request merged**